rpmbuild/copr_rpmbuild/builders/mock.py

```diff
@@ -27,6 +27,11 @@ def __init__(self, task, sourcedir, resultdir, config):
         self.buildroot_pkgs = task.get("buildroot_pkgs")
         self.enable_net = task.get("enable_net")
         self.repos = task.get("repos")
+        if self.repos and task.get("repo_priority") is not None:
+            repo_priority = task.get("repo_priority")
+            for repo in self.repos:
+                repo["priority"] = repo_priority
```

Comment:
This doesn't look right. The `self.repos` should contain all the repositories enabled for the Copr project (both the Copr repo itself but also external repos). We IMHO don't want to modify those.

I think we want to specify the priority here
https://github.com/fedora-copr/copr/blob/main/frontend/coprs_frontend/coprs/logic/complex_logic.py#L473-L487